x/projects/keeper/migrations.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	v3 "github.com/lavanet/lava/x/projects/migrations/v3"
```

Comment:
I think you meant "v4" here (the subdir names the target version);
But we should combine this anyhow into v2 -> v3 migration, so this ends up being ok :)

Reply:
done